ddtrace/internal/_encoding.pyx

```diff
@@ -165,8 +166,7 @@ cdef class Packer(object):
                        if ret != 0: break
                        ret = self._pack(v)
                        if ret != 0: break
-            elif PyList_CheckExact(o):
-                # Expect a list of traces or a list of spans
+            elif PyList_Check(o):
```

Comment:
@Kyle-Verhoog didn't we need this as CheckExact? I feel like we made that change at some point?

Reply:
Whatever the reason was, it doesn't seem to have a test case 😜